CHANGELOG.md

```diff
@@ -17,11 +17,12 @@
 * [pr#3397](https://github.com/chef/chef/pull/3397): Validate owner exists in directory resources
 * [pr#3418](https://github.com/chef/chef/pull/3418): Add `shell_out` mixin to Chef::Resource class for use in `not_if`/`only_if` conditionals, etc.
 * [pr#3406](https://github.com/chef/chef/pull/3406): Add wide-char 'Environment' to `broadcast_env_change` mixin for setting windows environment variables
-* [pr#3442](https://github.com/chef/chef/pull/3442): Add `resource_name` to top-level Resource class to make defining resources easier.  `use_automatic_resource_name` also added, to support convention-based resource naming.
+* [pr#3442](https://github.com/chef/chef/pull/3442): Add `resource_name` to top-level Resource class to make defining resources easier.
```

Comment:
This changelog entry might be confusing as it looks like that PR was closed and not merged?